woocommerce/woocommerce-android
# [Issue 10695] Fix Button Text Cut Off on 'Orders' Screen

Closes: #10695


### Description

This PR addresses the layout issue on the 'Orders' screen where the 'Troubleshooting' button text was previously cut off. The changes involve updating the layout constraints in the XML to ensure the button is fully visible and correctly positioned relative to the surrounding UI elements.

### Testing instructions

1. Go to 'Orders' screen.
2. Simulate an error. You can use this patch. 
[shows_the_notice_card.patch](https://github.com/woocommerce/woocommerce-android/files/14188379/shows_the_notice_card.patch)
3. Observe the top section of the screen where the error message "We couldn't load your data..." is displayed.
4. Notice that the Troubleshooting word isn't cut off.

### Images/gif



<kbd><img src="https://github.com/woocommerce/woocommerce-android/assets/1509205/b01a48f1-2c51-4f25-8236-dd005861c5aa" width="320"></kbd>
